src/@next/Tag/Tag.tsx

```diff
@@ -14,23 +14,37 @@ export interface TagProps extends React.HTMLAttributes<HTMLDivElement> {
   value?: string;
   onRemove?: (() => void) | null;
   textColor?: string;
+  disabled?: boolean;
 }
 
 export type TagRemoveContainerProps = React.HTMLAttributes<HTMLDivElement>;
 
 export type TagContentProps = React.HTMLAttributes<HTMLSpanElement> & TagProps;
 
 export const Tag = React.forwardRef<HTMLDivElement, TagProps>(function Tag(
-  { children, onRemove, value, textColor, ...props }: TagProps,
+  {
+    children,
+    onRemove,
+    value,
+    textColor,
+    onClick,
+    disabled,
+    ...props
+  }: TagProps,
   ref
 ) {
+  const handleTextColor = () => {
+    if (disabled) {
+      return Neutral.B85;
+    } else if (textColor) {
```

Comment:
there's no need else here if the first if is inverted, just an if will do

Reply:
Fixed